Symfony/CS/Tests/Fixer/Contrib/PhpUnitConstructFixerTest.php

```diff
@@ -23,76 +23,14 @@ class PhpUnitConstructFixerTest extends AbstractFixerTestBase
      */
     public function testFix($expected, $input = null)
     {
-        $fixer = $this->getFixer();
-
-        $fixer->configure(array(
```

Comment:
merge gone wrong from changes of weeks ago... I'll revert sorry I missed this one before opening the PR